packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -11,6 +11,17 @@
                 </div>
                 <div class="slds-col slds-order_1 slds-large-order_2 slds-size_1-of-1 slds-large-size_9-of-12">
                     <c-quantic-search-box engine-id={engineId}></c-quantic-search-box>
+                    <ul class="slds-tabs_default slds-tabs_default__nav" role="tablist">
+                        <c-quantic-tab label="All" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Articles" expression="@sfkbid" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="WebHelp" expression="@objecttypename==WebHelp" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Issues" expression="@jisourcetype AND NOT @jidocumenttype=&quot;WorkLog&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Community" expression="@objecttype==&quot;Message&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Closed Cases" expression="@objecttype==Case AND @sfisclosed==&quot;true&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Files" expression="@boxdocumenttype==File OR @spcontenttype==Document" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Confluence" expression="@connectortype==Confluence2Crawler" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Sharepoint" expression="@connectortype==SharePoint OR @connectortype==SharepointCrawler" engine-id={engineId}></c-quantic-tab>
```

Comment:
Are these tabs coming from Speedbit? Or from the mockup?

Reply:
its from the speedbit. I knew the query expressions for those. I'll change it as per the mockup